specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json

```diff
@@ -525,6 +550,15 @@
           "description": "Last modified time of this blueprint definition."
         }
       }
+    },
+    "WhoIsBlueprintContract":{
```

Comment:
>Contract [](start = 19, length = 8)

Should we call it WhoIsBlueprintResponse?

Reply:
We chatted on this offline. Keeping it as it is.